andrew-oyanguren/typescript-portfolio
# PR-16 About Page - Cards

### Summary

This work adds about cards to the AboutPage.

### Changes

- Conditional to render the Drawer component on small screens.
- New `aboutCardConstants` including `ABOUT_CARDS_CONFIG`.
- New **AboutCard** component, added to the AboutPage.
- Updated various styles to use new global CSS constants.

### Testing Guide

1. `npm run dev`
2. Navigate to About -> Verify 3 new cards.

https://github.com/andrew-oyanguren/typescript-portfolio/assets/69892684/e845a2db-c1aa-4937-bda7-37fe596ea5f5
